src/reaction_network/network.cpp

```diff
@@ -22,6 +22,11 @@
 #include <type_traits> // is_same<>
 #include <algorithm> // lexicographical_compare(), sort()
 #include <limits> // numeric_limits
+#include <ctime>
+
+#define TIME_DIFF(Tstart, Tend ) \
+    ((double) (1000000000L * ((Tend).tv_sec  - (Tstart).tv_sec) + \
+                              (Tend).tv_nsec - (Tstart).tv_nsec) / 1000000000L)
```

Comment:
If this is a temporary instrumentation for measuring reaction rate calculation performance, this should be removed.

Reply:
Done